libraries/core_libs/network/src/tarcap/packets_handlers/pbft_sync_packet_handler.cpp

```diff
@@ -24,6 +24,29 @@ PbftSyncPacketHandler::PbftSyncPacketHandler(std::shared_ptr<PeersState> peers_s
       network_sync_level_size_(network_sync_level_size),
       delayed_sync_events_tp_(1, true) {}
 
+void PbftSyncPacketHandler::validatePacketRlpFormat(const PacketData &packet_data) {
+  checkPacketRlpList(packet_data);
+
+  // TODO: the way we handle PbftSyncPacket must be changed as we are using empty rlp as valid data -> it means pbft
```

Comment:
OT: I have an idea, maybe instead of writing TODO in our code, we should first create task in github and do something like this: `TODO[task_id] ... ` so we know it should be done and it is easy to search where is should be changed

Reply:
again, very good point :)